checker/bin-devel/javac-debug

```diff
@@ -24,12 +24,11 @@ binaryDir="${mydir}"/../dist
 javacJar=${binaryDir}/javac.jar
 
 cfDir="${mydir}"/../..
-annoToolsDir="${cfDir}"/../annotation-tools
 stubparserDir="${cfDir}"/../stubparser
 classes="build/classes/java/main"
 resources="build/resources/main"
 
-buildDirs="${cfDir}"/dataflow/"${classes}":"${cfDir}"/javacutil/"${classes}":"${cfDir}"/framework/"${classes}":"${cfDir}"/framework/"${resources}":"${cfDir}"/checker/"${classes}":"${cfDir}"/checker/"${resources}":"${cfDir}"/checker-qual/"${classes}":"${stubparserDir}"/javaparser-core/stubparser.jar:"${annoToolsDir}"/scene-lib/bin:"${annoToolsDir}"/annotation-file-utilities/annotation-file-utilities-all.jar:$( (cd "${cfDir}" && ./gradlew -q :checker:printPlumeUtilJarPath) )
+buildDirs="${cfDir}"/dataflow/"${classes}":"${cfDir}"/javacutil/"${classes}":"${cfDir}"/framework/"${classes}":"${cfDir}"/framework/"${resources}":"${cfDir}"/checker/"${classes}":"${cfDir}"/checker/"${resources}":"${cfDir}"/checker-qual/"${classes}":"${stubparserDir}"/javaparser-core/stubparser.jar:$( (cd "${cfDir}" && ./gradlew -q :checker:printClasspath) )
```

Comment:
Is it not possible to use the jar file because of shading, or for some other reason?